src/appComponents/sessions/SessionsController.js

```diff
@@ -8,11 +8,13 @@ var sessionsModule = angular.module('oe.sessions', ['ui.router']);
 /* ----------------------------- Sessions List ---------------------------- */
 var singleSession = {};
 sessionsModule.config(['$stateProvider', function($stateProvider) {
+    
     $stateProvider.state('sessions', {
-        url: '/sessions',
-        templateUrl: 'appComponents/sessions/sessions.html',
-        controller: 'SessionsController'
+      "url": '/sessions',
+      "templateUrl": 'appComponents/sessions/sessions.html',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-webapp/file/2431610224/issues/source?bid=3233857&fileBranchId=3238420#l14)